### ISSUE TITLE
TFTP Improper Data Retransmission

### ISSUE THREAD

**Issue by pbertera** [psychomario/PyPXE#98]
When we a data block is retransmitted the file pointer isn't moved back of one block size.
This should solve:

```python
                # if we haven't recieved an ACK in timeout time, retry
                for client in self.ongoing:
                    if client.no_ack():
                        self.logger.warning("Retransmission of block {0}".format(client.block))
                        client.fh.seek(-client.blksize, 1)
                        client.send_block() 
```

**Comment by psychomario** [comment]
I think it would be cleaner to always seek to `blksize*block` just before we `read`.

Can you test and submit a PR for this? If not I should be able to do so tomorrow.

**Comment by psychomario** [comment]
I believe this was fixed in #100

**Comment by mmattioli** [comment]
@psychomario, this should technically remain open until the fix is merged into `master`. As long as it's in the commit message which merges `development` into `master` we're fine, it'll automatically close this.
